interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1585,16 +1586,18 @@ def convert(filename=None, output_name=None, text=None):
 
     try:
         if filename is not None:
-            with filename.open('r', encoding='latin-1') as f:
-                code = compile(f.read(), str(filename), 'exec')
+            with filename.open("r", encoding=encoding) as f:
+                text = f.read()
```

Comment:
I think you can get this to one line 😉 
```suggestion
            text = filename.read_text(encoding=encoding)
```